docker-compose.yml

```diff
@@ -17,3 +19,21 @@ services:
       - ./static:/static
     depends_on:
       - db
+  celery:
+    build: .
+    command: celery -A config worker -l INFO
+    env_file: .env
+    volumes:
+      - .:/code
+    depends_on:
+      - db
+      - redis
+  celery-beat:
+    build: .
+    command: celery -A config beat -l INFO
+    env_file: .env
+    volumes:
+      - .:/code
+    depends_on:
+      - db
+      - redis
```

Comment:
missing empty line on end of file

Reply:
You can add flower to have visibility of what happened to tasks:
https://flower.readthedocs.io/en/latest/